oop/tests/unit/router/routes.spec.js

```diff
@@ -0,0 +1,54 @@
+import { isPastPath as testFunction } from "@/router/routes.js";
```

Comment:
`isPastPath` shouldn't need an alias here also.

Reply:
Whoops! Sorry I missed that one. Fixed in 04456a96fc65ac38d41e817a284992d1bd7e9c22